src/Sarif.UnitTests/SarifVersionOneToCurrentTests.cs

```diff
@@ -1114,5 +1119,325 @@ public void SarifTransformerTests_ToCurrent_BasicResult()
+              ""default"": ""The  value of the '{0}' attribute is not quoted. Wrap the attribute value in single or double quotes.""
+            }
+          },
+          ""WEB1066.TagNameIsNotLowercase"": {
+            ""id"": ""WEB1066"",
+            ""shortDescription"": ""The tag name is not lowercase."",
+            ""messageFormats"": {
+              ""default"": ""Convert the name of the <{0}> tag to lowercase.""
+            }
+          }
+        }
+      }
+    ]
+  }";
+
+            SarifLog v2Log = TransformVersionOneToCurrent(v1LogText);
+
+            string v2LogText = JsonConvert.SerializeObject(v2Log, SarifTransformerUtilities.JsonSettingsV2);
+            string v2LogExpectedText =
+@"{
+  ""$schema"": ""http://json.schemastore.org/sarif-2.0.0"",
+  ""version"": ""2.0.0"",
+  ""runs"": [
+    {
+      ""tool"": {
+        ""name"": ""CodeScanner""
+      },
+      ""results"": [
+        {
+          ""ruleId"": ""WEB1079"",
+          ""message"": {
+            ""arguments"": [
+              ""shape""
+            ]
+          },
+          ""ruleMessageId"": ""default"",
+          ""locations"": [
+            {
+              ""physicalLocation"": {
+                ""fileLocation"": {
+                  ""uri"": ""http://localhost:34420/HtmlFixes.html""
+                },
+                ""region"": {
+                  ""startLine"": 20,
+                  ""startColumn"": 69,
+                  ""endColumn"": 74,
+                  ""offset"": 720,
+                  ""length"": 5,
+                  ""snippet"": {
+                    ""text"": ""<area alt=\""Here is some text\"" coords=\""10 20 20\"" href=\""moon.html\"" shape=circle xweb:fixindex=\""0\"" />""
+                  }
+                }
+              }
+            }
+          ],
+          ""fixes"": [
+            {
+              ""description"": {
+                ""text"": ""Wrap attribute values in single quotes.""
+              },
+              ""fileChanges"": [
+                {
+                  ""fileLocation"": {
+                    ""uri"": ""http://localhost:34420/HtmlFixes.html""
+                  },
+                  ""replacements"": [
+                    {
+                      ""offset"": 720,
+                      ""insertedBytes"": ""Jw==""
+                    },
+                    {
+                      ""offset"": 725,
+                      ""insertedBytes"": ""Jw==""
+                    }
+                  ]
+                }
+              ]
+            },
+            {
+              ""description"": {
+                ""text"": ""Wrap attribute value in double quotes.""
+              },
+              ""fileChanges"": [
+                {
+                  ""fileLocation"": {
+                    ""uri"": ""http://localhost:34420/HtmlFixes.html""
+                  },
+                  ""replacements"": [
+                    {
+                      ""offset"": 720,
```

Comment:
In v2, a `replacement` object looks like this (§3.40, replacement object):
```
"fileChanges": [
  {
    "deletedRegion": {
      "offset": 12,
      "length": 5
    },
    "insertedContent": {
      "binary": "ZXhhbXBsZQ=="
    }
  },
  {
    "deletedRegion": {
      "offset": 20,
      "length": 3
    }
  },
  {
    "deletedRegion": {
      "offset": 312,
      "length": 0
    },
    "insertedContent": {
      "binary": "ZXhhbXBsZQ=="
    }
  }
]
```
Are we not up to date on that schema change?

Reply:
Must not be.

Reply:
Confirmed. Want me to include those changes here?

Reply:
Or I can do them separately and merge.

Reply:
Let's do this, in sequence:
1) I'll review this the way it is, and we'll merge it.
2) Separate PR: Update the schema and modify the fix transformation logic.

Reply:
Oh, and
    3. Look for other missing schema changes. I could've sworn we were up to date.

Reply:
Sounds good. I'll take 5 minutes to look back and see if it was ever changed,